### DISCUSSION TITLE
Feature Request: Ironing Inset Setting

### DISCUSSION THREAD

**Discussion by arobdog** [OrcaSlicer/OrcaSlicer#2778] [Ideas]
I have encountered an issue with the rectilinear ironing pattern where some filament is pushed over the edge of the top surface slightly causing a 'feathering' pattern around the edge of the layers. This can be mitigated to some extent by reducing the extrusion of material with either the filament 'Flow ratio' or with the 'Ironing flow' settings. However reducing this can reduce the feathering but it can also negatively effect the top surface finish/quality of ironing.

I propose that an 'Ironing inset' setting be added to Orca Slicer which would allow this issue to be offset and potentially fixed altogether. This setting takes a number value and will inset the ironing pattern on the top surfaces by this amount from the outer wall. This setting exists in Cura slicer already and should be implemented in Orca Slicer too.

(Concentric ironing does not have this issue, however the surface finish of concentric is not ideal for all circumstances and the use cases I have in particular. I find it has worse consistency when printing a wide variety of objects and more complex top surfaces.)

**Comment by SirNukalot** [comment]
Agreed, definitly one of the things i'm missing the most since switching from cura.

**Comment by sadmuffins** [comment]
This is exactly what I came here to look for. I just switched to Orca and was confused that I couldn't find ironing inset. Definitely hope this feature is added!

**Reply by dfosantos** [reply]
Here you go https://github.com/SoftFever/OrcaSlicer/pull/7969

**Reply by henrivdr** [reply]
And just like that, @sadmuffins is no longer sad :-)
Thanks @dfosantos

**Reply by sadmuffins** [reply]
Yay! Thank you @dfosantos

**Comment by dfosantos** [comment]
https://github.com/SoftFever/OrcaSlicer/pull/7969

**Comment by LocalBug24** [comment]
Forgive me, I'm very new to github and orca. I'm looking for this feature - how do I access it, i'm using v. 2.2.0?

**Reply by dfosantos** [reply]
You need to download the nightly build, as this feature is still not in the stable release.
https://github.com/SoftFever/OrcaSlicer/releases

**Reply by LocalBug24** [reply]
Thank you :)
